templates/repo/projects/view.tmpl

```diff
@@ -2,12 +2,15 @@
 <div class="page-content repository">
 	{{template "repo/header" .}}
 	<div class="ui container">
-		<div class="ui three column stackable grid">
+		<div class="ui two column stackable grid">
 			<div class="column">
 				{{template "repo/issue/navbar" .}}
 			</div>
 			<div class="column right aligned">
 				{{if and .CanWriteProjects (not .Repository.IsArchived) .PageIsProjects}}
+					<a class="ui green button show-modal item" href="{{$.RepoLink}}/issues/new?project={{$.Project.ID}}">{{.i18n.Tr "repo.issues.new"}}</a>
+				{{end}}
+				{{if and (not .Repository.IsArchived) .PageIsProjects}}
 					<a class="ui green button show-modal item" data-modal="#new-board-item">{{.i18n.Tr "new_project_board"}}</a>
```

Comment:
Why remove the `.CanWriteProjects` condition here?

Reply:
I was thinking you can add an issue on a project even if you can write a project, after verification I was wrong. I fix that ;)

Reply:
Ok, the `if` blocks now have the same condition, so you can move the buttons below one shared block.